service-front/module/Application/src/Model/Service/Lpa/Communication.php

```diff
@@ -23,66 +24,59 @@ class Communication extends AbstractEmailService
      * @var Container
      */
     private $userDetailsSession;
+    private $emailTemplateRef;
+    private $data;
+    private $to;
+    private $lpaTypeTitleCase;
+    private $userEmailAddress;
 
     public function sendRegistrationCompleteEmail(Lpa $lpa)
     {
         // Get the signed in user's email address.
-        $userEmailAddress = $this->userDetailsSession->user->email->address;
-        $to = [$userEmailAddress];
+        $this->userEmailAddress = $this->userDetailsSession->user->email->address;
```

Comment:
I would avoid setting private member variables on a class unless I needed the state in multiple places. Even then, I would pass the state to the methods/functions which need it, rather than set member variables, unless I ended up with long argument lists.

Some of these are warranted, but I don't see any reuse of `$this->userEmailAddress` or `$this->to`, for example.

Reply:
Both userEmailAddress and to are used in 2 places. I just made them member variables to avoid a load of passing arguments around.

Reply:
I can pass userEmailAddress around though. will change this

Reply:
OK , have changed userEmailAddress.  Have attempted to change $to, but it isn't very pretty because gets passed around. Might be better is $to stayed as a private variable

Reply:
Now fixed